cartridge-cli.lua

```diff
@@ -922,19 +919,58 @@ ENV PATH="/usr/share/tarantool/tarantool-enterprise:${"$"}{PATH}"
 -- * ---------------- Generic packing ----------------
 
 local function get_rock_versions(project_dir)
-    local query = luarocks_searcher.make_query("")
-    query.exact_name = false
+    local function read_manifest_file(filepath)
+        local res = {}
+
+        local file_content = read_file(filepath)
+        file_content = file_content:gsub("^#![^\n]*\n", "")
+
+        local chunk, ok, err
+        if _VERSION == "Lua 5.1" then -- Lua 5.1
+            chunk, err = loadstring(file_content, filepath)
+            if chunk then
+                setfenv(chunk, res)
+                ok, err = pcall(chunk)
+            end
+        else -- Lua 5.2
+            chunk, err = load(file_content, filepath, "t", res)
+            if chunk then
+                ok, err = pcall(chunk)
+            end
+        end
+        if not chunk or not ok then
+            return nil, string.format('Failed to run file %s: %s', filepath, err)
+        end
+
+        return res
+    end
 
-    local manifest_path = luarocks_path.rocks_dir(fio.pathjoin(project_dir, '.rocks'))
+    local dependencies = {}
+    -- XXX: fix manifest filepath compution
+    local manifest_filepath = fio.pathjoin(project_dir, '.rocks/share/tarantool/rocks/manifest')
 
-    local results = {}
-    luarocks_searcher.manifest_search(results, manifest_path, query)
+    if fio.path.exists(manifest_filepath) then
```

Comment:
Shouldn't we check that it's a file? (it could be a user created function, by mistake)

Reply:
I added a check.